packages/amplify-cli/scripts/post-install.js

```diff
@@ -13,6 +14,11 @@ try {
 } catch (e) {
   // could not delete the cache directory but don't want to fail the installation
 }
+
+// execute post-install amplify command
+const ampPath = path.resolve(path.join(__dirname, '..', 'bin', 'amplify'));
```

Comment:
what would be the __dirname in this case and is the amplify bin always inside the current dir's bin dir?

Reply:
__dirname is the absolute path of the current file at runtime. So this path points to the bin folder of the amplify-cli package which will always have the amplify bin file